include/wil/cppwinrt_authoring.h

```diff
@@ -198,6 +192,23 @@ std::vector<unique_com_class_object_cookie> register_com_server(
     return registrations;
 }
 
+template <typename... Types>
+struct clsid_array
+{
+    std::array<GUID, sizeof...(Types)> value;
+    explicit clsid_array(std::array<GUID, sizeof...(Types)> value) : value(std::move(value))

```

Comment:
Note that the pattern here is "copy then move". In practice this value is always move-constructed from temporary, or in more recent standard just materialized without any temporary.

Reply:
`std::move` is pointless here. It's a `std::array` and `GUID`s. Accept by `const` reference and just assign and make it `constexpr` while you are at it.